assets/css/_schedule-page-line-diagram.scss

```diff
@@ -124,10 +124,10 @@
 .m-schedule-diagram__connections {
   @include icon-size-inline($base-spacing);
 
-  a {
-    margin-right: calc(#{$base-spacing} / 4);
-    vertical-align: top;
-  }
+  display: inline-flex;
+  flex-wrap: wrap;
+  gap: calc(#{$base-spacing} / 4);
+  line-height: initial;
```

Comment:
I'm a little surprised that something like `align-items: center` wasn't necessary 😳